python/test/test_hail/expr/test_expr.py

```diff
@@ -272,6 +272,15 @@ def test_aggregators_linreg(self):
         self.assertAlmostEqual(r.p_value[1], 0.63310173)
         self.assertAlmostEqual(r.n, 5)
 
+    def test_aggregators_virtualds(self):
+        xs = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+        ys = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```

Comment:
You might scramble and/or shift the x and y values to check errors mixing up the order of x and y.